src/presenters/title-presenter.php

```diff
@@ -28,14 +35,27 @@ class Title_Presenter extends Abstract_Indexable_Tag_Presenter {
 	 *
 	 * @return string The raw value.
 	 */
-	public function get() {
+	public function refresh() {
 		// This ensures backwards compatibility with other plugins using this filter as well.
 		\add_filter( 'pre_get_document_title', [ $this, 'get_title' ], 15 );
 		$title = \wp_get_document_title();
 		\remove_filter( 'pre_get_document_title', [ $this, 'get_title' ], 15 );
 		return $title;
 	}
 
+	/**
+	 * Returns a tag in the head.
+	 *
+	 * @return string The tag.
+	 */
+	public function present() {
```

Comment:
@increddibelly Maybe I'm missing something, but why do we have to extend `present()` here? Is it necessary or we can remove to avoid code duplication?

Reply:
this is the only presenter that has a single variable in its template tag :-)